api/gamerunner.js

```diff
@@ -0,0 +1,42 @@
+let _ = require("lodash");
+let lovebotPlayer = require("lovebotplayer");
+let Tournament = require('../model/game/tournament');
+let HttpProxyStrategy = require('../httpproxy');
+
+// Called by the gamerunner api controller
+module.exports = {
+  runGame: runGame
+};
+
+function runGame(request) {
```

Comment:
If its only one function, I usually just do module.exports = () => {}, so you can just call the thing with one less dot.